### PR TITLE
Fix ResolveToParents, which seem to broke https://github.com/TiVo/intellij-haxe/issues/196 Also it used to throw error on loading of my test project

### PR DESCRIPTION
**Pull request by as3boyan**
Fix ResolveToParents, which seem to broke https://github.com/TiVo/intellij-haxe/issues/196
Also it used to throw error on loading of my test project

### CODE REVIEW

File: src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java [hunk 1]
```diff
@@ -90,6 +91,10 @@ public boolean isSoft() {
   }
 
   private List<? extends PsiElement> resolveNamesToParents(List<? extends PsiElement> nameList) {
+    if (nameList == null) {
```
Comment (as3boyan) [inline src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java]: cachedNames from HaxeResolver if I recall correctly returns null on project load (tested on my test project)

Comment (as3boyan) [inline src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java]: From here https://github.com/TiVo/intellij-haxe/blob/FixResolveToParentsForCaseWhenCachedNamesIsNull/src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java#L163

**Comment by as3boyan** [comment]
Yay tests passed

**Comment by EBatTiVo** [comment]
Approved.  Are you intending to make this a release or just check in with added release notes?

**Comment by as3boyan** [comment]
When @fzzr- will test if it fixes his issues, then I think it might be a good idea to make a quick release, because issue is pretty critical

**Comment by EBatTiVo** [comment]
OK.  I am adding a new section to CONTRIBUTING.md that describes the release process.  It's actually a good bit of work to do it right.

**Comment by EBatTiVo** [comment]
The release process has been documented and pushed to master: 1b9652cf3aa33ecb3bd1d7a48b26675314503821

**Pull request merged**